plugin.rb

```diff
@@ -14,113 +14,59 @@
 
 register_asset "stylesheets/common/discourse-automation.scss"
 
-require_relative "app/lib/discourse_automation/triggerable"
-require_relative "app/lib/discourse_automation/scriptable"
-require_relative "app/core_ext/plugin_instance"
-
-after_initialize do
-  module ::DiscourseAutomation
-    PLUGIN_NAME = "discourse-automation"
+module ::DiscourseAutomation
+  PLUGIN_NAME = "discourse-automation"
 
-    CUSTOM_FIELD = "discourse_automation_ids"
-    TOPIC_LAST_CHECKED_BY = "discourse_automation_last_checked_by"
-    TOPIC_LAST_CHECKED_AT = "discourse_automation_last_checked_at"
+  CUSTOM_FIELD = "discourse_automation_ids"
+  TOPIC_LAST_CHECKED_BY = "discourse_automation_last_checked_by"
+  TOPIC_LAST_CHECKED_AT = "discourse_automation_last_checked_at"
+end
 
-    class Engine < ::Rails::Engine
-      engine_name PLUGIN_NAME
-      isolate_namespace DiscourseAutomation
-    end
-  end
+require_relative "app/core_ext/plugin_instance"
+require_relative "lib/discourse_automation/engine"
+require_relative "lib/discourse_automation/scriptable"
+require_relative "lib/discourse_automation/triggerable"
 
+after_initialize do
   %w[
-    app/controllers/admin/discourse_automation/admin_discourse_automation_automations_controller
-    app/controllers/admin/discourse_automation/admin_discourse_automation_controller
-    app/controllers/admin/discourse_automation/admin_discourse_automation_scriptables_controller
-    app/controllers/admin/discourse_automation/admin_discourse_automation_triggerables_controller
```

Comment:
Looks like those got lost, hence the spec failure? 😃

Reply:
That should not be necessary because they are autoloaded. Unfortunately, the names did not match, but I think I fixed it.